SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationButton.java

```diff
@@ -31,6 +35,11 @@ public ConversationButton(Context context, ButtonControl model, int defStyle) {
         initBackgroundColorStates();
         setLines(1);
         setEllipsize(TextUtils.TruncateAt.END);
+        if (conversationVersion >= 4) { // text/font styling added in v4
```

Comment:
Yeah, I thought about this, and this is a SwrveConversationButton, that already takes a ButtonControl object so passing in a conversationversion isn't such a big deal here IMO.

It needs to be able to render old style of conversations and new ones, hence passing the version in was needed. 

This is WIP, as I'm removing the default styles and applying them programmatically so all configuration is in one place. More to come.....